NetCord/MessagePollProperties.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.Json.Serialization;
+
+namespace NetCord;
+
+public partial class MessagePollProperties
+{
+    [JsonPropertyName("question")]
+    public MessagePollMediaProperties Question { get; set; }
+
+    [JsonPropertyName("answers")]
+    public MessagePollAnswerProperties[] Answers { get; set; }
+
+    public MessagePollProperties(MessagePollMediaProperties question, MessagePollAnswerProperties[] answers)
+    {
+        Question = question;
+        Answers = answers;
+    }
```

Comment:
You can use primary constructor here, also the value of answers and the corresponding property should be `IEnumerable<MessagePollAnswerProperties>`